### PR TITLE
Added utility for generating a pkg-config file

### PR DESCRIPTION
**Pull request by harlanhaskins**

### CODE REVIEW

**Review by CodaFi — APPROVED** [review]
💥

**Comment by harlanhaskins** [comment]
This is untested on Linux, but this should cover the case of "we need to make a pkg-config file for every distro?!"

**Comment by CodaFi** [comment]
I'll update #13 with it

**Comment by harlanhaskins** [comment]
Don't merge quite yet. I'm updating the README.

**Comment by harlanhaskins** [comment]
Merge if you approve the latest README changes

File: README.md [hunk 1]
```diff
@@ -44,13 +44,15 @@ Plus, it provides common wrappers around oft-used types like `Function`, `Global
...
-
-```
-brew install llvm
-```
+build.
+
+- Install LLVM 3.9 using your favorite package manager. For example:
+  - `brew install llvm`
+- Ensure `llvm-config` is in your `PATH`
+  - That will reside in the `/bin` folder wherever your package manager
```
Comment (CodaFi) [inline README.md:52]: I don't think this is all that helpful, users don't really need to know this location to build.  Maybe mention invoking `which llvm-config` or something.

Comment (harlanhaskins) [reply README.md:52]: It helped me because it didn't end up in my path after installing from homebrew :sad:

Comment (CodaFi) [reply README.md:52]: Oh, I see.  Carry on.

**Comment by CodaFi** [comment]
:shipit: 

⛵️

**Comment by harlanhaskins** [comment]
Now let's get #13 working!!!

**Pull request merged**